index.html

```diff
@@ -5618,17 +5618,17 @@ <h2>Interlacing</h2>
     <section id="12Filter-selection">
       <h2>Filter selection</h2>
 
-      <p>For images of <a>colour type</a> 3 (indexed-colour), filter type 0 (None) is usually the most effective. Colour images
-      with 256 or fewer colours should almost always be stored in <a>indexed-colour</a> format; <a>truecolour</a> format is likely to be
+      <p>For images of <a>color type</a> 3 (indexed-color), filter type 0 (None) is usually the most effective. Colour images
```

Comment:
```suggestion
      <p>For images of <a>color type</a> 3 (indexed-color), filter type 0 (None) is usually the most effective. Color images
```
Maybe do another search that's case-insensitive.